src/popup/pages/Notifications.vue

```diff
@@ -74,12 +57,13 @@ const DIR_SUPERHERO = 'superhero';
 export default defineComponent({
   name: 'Notifications',
   components: {
+    InfiniteScroll,
     NotificationItem,
-    Tabs,
-    Tab,
   },
   setup(props, { root }) {
     const direction = ref(DIR_ALL);
+    const notificationToShow = ref<INotification[]>([]);
```

Comment:
what do you thing about using plural form (as we display multiple notifications and not just one)?